tests/func/test_cli.py

```diff
@@ -206,3 +212,42 @@ def test_unknown_subcommand_help(capsys):
     captured = capsys.readouterr()
     help_output = captured.out
     assert output == help_output
+
+
+def test_similar_command_suggestions(caplog):
+    suggestions = {
+        "addd": "add",
+        "stats": "status",
+    }
```

Comment:
This could be unified with `@pytest.mark.parametrize`.

Reply:
pytest is still new to me but awesome that this feature exists. Updated the tests to use this 👍